components/incoming/CorporationInternship.vue

```diff
@@ -14,7 +14,7 @@
 .corporation-internship-container {
   width: 100%;
   height: 100vh;
-  background-image: url('../assets/icx-corporation-internship.jpg');
+  background-image: url(`@/assets/pages/icx-corporation-internship.jpg`);
```

Comment:
```suggestion
  background-image: url("~@/assets/images/icx-corporation-internship.jpg");
```
- パスですが、assets/pages/ではなく、assets/images/でお願いします
- @の前に~つけるのと、ダブルクォーテーション(シングルクォーテーションでも行けそう？)で囲んでください。
バッククォートだとうまく動きませんでした

Reply:
質問なのですが、

画像ファイルのパス表記ですが、
Styleでは"~@/assets/images/ファイル名"
Templateでは"@/assets/images/ファイル名"
でうまくいくのではないかと思います。

styleとtemplateではパスが少し違うのはなんでなのかな、、と